File: src/main/java/com/github/codingvampyre/drachenfest/dtos/ParticipantCreateRequestDto.java
package com.github.codingvampyre.drachenfest.dtos;

import lombok.Data;

import javax.validation.constraints.*;

@Data
public class ParticipantCreateRequestDto {
    @NotEmpty
    @Email
    private String email;

    @NotEmpty
    @NotEmpty
    @Size(min = 8)
    private String password;
}

File: src/main/java/com/github/codingvampyre/drachenfest/dtos/ParticipantCreateRequestDto.kt
package com.github.codingvampyre.drachenfest.dtos

import javax.validation.constraints.Email
import javax.validation.constraints.NotEmpty
import javax.validation.constraints.Size

class ParticipantCreateRequestDto (
		@field:NotEmpty
		@field:Email
        val email: String,

		@field:NotEmpty
		@field:Size(min = 8)
        val password: String,
)
